content/support.md

```diff
@@ -0,0 +1,23 @@
++++
+title = "Support"
+description = "Support for deploying and using KEDA"
++++
+
+## OSS support
+KEDA is an open-source project under CNCF Foundation and provides best-effort to support its community of users.
+
+We have a nice community that is always happy to help each other. We support our users either via GitHub discussions or [#Keda](https://kubernetes.slack.com/messages/CKZJ36A5D) channel on the [Kubernetes slack](https://slack.k8s.io/). We use GitHub for tracking bugs and feature requests.
+
+## Kubernetes support
+The supported window of Kubernetes versions with KEDA is known as "N-2" which means that KEDA will provide support for running on N-2 at least.
+
+However, maintainers can decide to extend this by supporting more minor versions based on the required CRDs being used; but there is no guarantee.
+
+> Example - At time of writing, Kubernetes 1.19 is the latest minor version so KEDA can only use new features that were introduced in 1.17
+
+You can learn more about the currently supported Kubernetes version in our [FAQ](https://keda.sh/docs/latest/faq/).
+
+## Commercial support
```

Comment:
I'm leaning towards showing this before Kubernetes support as this is more important, thoughts @JorTurFer @zroubalik ?

Reply:
Alright. would prioritize appropriately